examples/eks/eks_cluster_gitops/README.md

```diff
@@ -0,0 +1,26 @@
+## EKS and CAST AI example for GitOps onboarding flow
+
+Following example shows how to onboard EKS cluster to CAST AI using GitOps flow.
+In GitOps flow CAST AI components such as `castai-agent`, `castai-cluster-controller`, `castai-evictor`, `castai-spot-handler`, `castai-kvisor` 
+are installed by the user outside of Terraform using other means e.g. ArgoCD.
+
+Steps to take to successfully onboard EKS cluster to CAST AI using GitOps flow:
+Prerequisites:
+- CAST AI account
+- Obtained CAST AI [API Access key](https://docs.cast.ai/docs/authentication#obtaining-api-access-key) with Full Access
+1. Configure `tf.vars.example` file with required values. If EKS cluster is already managed by Terraform you could instead directly reference those resources.
+2. Run `terraform init`
+3. Run `terraform apply` and make a note of `cluster_id` and `cluster_token` output values. At this stage you would see that your cluster is in `Connecting` state in CAST AI console
+4. Install CAST AI components using Helm. Use `cluster_id` and `cluster_token` values to configure Helm releases:
```

Comment:
Can we add here an example of how to use argo/flux/helm (one of them)

Reply:
Thanks! Added